test/index.test.ts

```
@@ -525,3 +525,43 @@ describe("ofetch", () => {
 });
+
+describe("isJSONSerializable", () => {
+  let isJSONSerializable: any;
+
+  beforeAll(async () => {
+    const utils = await import("../src/utils.ts");
+    isJSONSerializable = utils.isJSONSerializable;
+  });
+
+  it("returns false for null", () => {
+    expect(isJSONSerializable(null)).toBe(false);
```

ESLint

[error] Use `undefined` instead of `null`.

(unicorn/no-null)